packages/firebase_admob/ios/Classes/FLTMobileAd.m

```diff
@@ -129,7 +151,7 @@ - (void)show {
 - (void)placeBannerPreIos11 {
   UIView *screen = [FLTMobileAd rootViewController].view;
   CGFloat x = screen.frame.size.width / 2 - _banner.frame.size.width / 2;
-  CGFloat y = screen.frame.size.height - _banner.frame.size.height;
+  CGFloat y = screen.frame.size.height - _banner.frame.size.height - _anchorOffset;
```

Comment:
No offset scaling on iOS? If that is correct, please ignore my comments about doing the scaling on the Dart side. I still think the offset value should be a `double` though.